app/models/user.rb

```diff
@@ -1,2 +1,12 @@
 class User < ApplicationRecord
+  has_many :posts
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+
+  def update_posts_count
+    self.posts_counter = posts.count
+    save
+  end
```

Comment:
- [ ]  The user model should have many comments and likes as well. As you know, it is the user who will create the comments and the likes, so you should have that relationship created as well.